samples/powershell/kafkapwsh/function.json

```diff
@@ -0,0 +1,18 @@
+{
+    "bindings": [
+      {
+        "direction": "in",
+        "type": "kafkaTrigger",
+        "consumerGroup": "azfunc",
+        "protocol": "saslSsl",
+        "authenticationMode": "plain",
+        "username": "<API-Key>",
+        "password": "",
+        "sslCaLocation": "",
```

Comment:
these are names for the secrets to be looked up, right?

Reply:
In this case, I use Confluent Cloud to provide my Infrastructure, these are named reference parameters to connect in Kafka cluster.